lightkurve/lightcurve.py

```diff
@@ -1227,7 +1240,7 @@ def to_pandas(self, columns=['time', 'flux', 'flux_err', 'quality',
         """
         return super(KeplerLightCurve, self).to_pandas(columns=columns)
 
-    def to_fits(self, path=None, overwrite=False, **extra_data):
+    def to_fits(self, path=None, overwrite=False, aperture_mask=None, **extra_data):
```

Comment:
All the data written by `to_fits` is held as class attributes.  I think we'll want the same with `aperture_mask`, i.e. I think we'll want `aperture_mask` to become an optional attribute that can be passed to the constructor of `LightCurve`?  If set, `to_fits` would then write `self.aperture_mask` as an optional extension.

Reply:
`aperture_mask` has the distinction of being the only metadata that is ndarray, and requires a special ImageHDU extension.  The `extra_data` keywords assume inputs are vectors or scalar/text, so `aperture_mask` will still require some special handling, but you're right that it need not be a standalone kwarg.

Reply:
I am going to leave `aperture_mask` as a new keyword to `to_fits`, to accelerate the current atomic pull request, and open issue #341 to add an optional `aperture_mask` class attribute to `LightCurve`.

Reply:
100% agreed!